membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -143,20 +143,20 @@ class TouchpointComponents(
   )
 
   lazy val futureCatalog: Future[CatalogMap] = catalogService.catalog
-    .map(_.fold[CatalogMap](error => { println(s"error: ${error.list.toList.mkString}"); Map() }, _.map))
+    .map(_.fold[CatalogMap](error => { logger.errorNoPrefix(scrub"error: ${error.list.toList.mkString}"); Map() }, _.map))
```

Comment:
not sure where the println would have gone (probably not cloudwatch), it should be using the logger.